lib/worker/fake-tty.js

```diff
@@ -2,19 +2,30 @@
 const tty = require('tty');
 const options = require('./options').get();
 
-if (options.tty) {
-	Object.assign(process.stdout, {isTTY: true}, options.tty);
+const fakeTTYs = [];
 
-	if (options.tty.colorDepth !== undefined) {
-		process.stdout.getColorDepth = () => options.tty.colorDepth;
+const isatty = tty.isatty;
+tty.isatty = fd => {
+	if (fakeTTYs.indexOf(fd) !== -1) {
+		return true;
 	}
+	return isatty(fd);
+};
 
-	const isatty = tty.isatty;
-	tty.isatty = function (fd) {
-		if (fd === 1 || fd === process.stdout) {
-			return true;
-		}
+const simulateTTY = (stream, options) => {
+	Object.assign(stream, {isTTY: true}, options);
```

Comment:
This assigns `colorDepth`. Better to destructure options and assign just `columns` and `rows`:

```js
const simulateTTY = (stream, {colorDepth, columns, rows}) => {
  Object.assign(stream, {isTTY: true, columns, rows})

  if (colorDepth !== undefined) {
```